### DISCUSSION TITLE
Dokku Documentation Styling Error on Chrome

### DISCUSSION THREAD

**Discussion by BTRBT** [dokku/dokku#5307] [General]
![image](https://user-images.githubusercontent.com/33989637/184937116-a242693c-0dde-46b9-8423-791fb86799d6.png)

Hopefully a picture is worth a thousand words.

Keep up the great work, guys. I love Dokku!

**Comment by josegonzalez** [comment]
This should be fixed now (and should have been reported as a bug ;) ).
